### PR TITLE
Implement reciprocal

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin @fdwr PTAL, thanks.

### CODE REVIEW

**Review by fdwr — APPROVED** [review]

File: test/unary_test.js [hunk 1]
```diff
@@ -736,4 +736,54 @@ describe('test unary', function() {
         ],
         [3, 2, 2, 1]);
   });
+
+  it('reciprocal', function() {
```
Comment (huningxin) [inline test/unary_test.js:740]: Consider supporting 0-D scalar input.

**Review by huningxin — APPROVED** [review]
LGTM with a comment, thanks!

**Review by fdwr — APPROVED** [review]

**Comment by huningxin** [comment]
@BruceDai , please resolve the merge conflict, so I can merge, thanks!

**Comment by BruceDai** [comment]
Fixed. Thanks @huningxin!

**Comment by BruceDai** [comment]
And once this pr merged, I will rebase #54, then ping you to help merge that. Thanks.

**Pull request merged**